index.html

```diff
@@ -1247,7 +1253,10 @@ <h3>Algorithm</h3>
                 <li class="changed">If <var>context</var> was previously dereferenced,
                   then the processor MUST NOT do a further dereference, and
                   <a>context</a> is set to the
-                  previously established <a>internal representation</a>.
+                  previously established <a>internal representation</a>,
```

Comment:
```suggestion
                  previously established <a>internal representation</a>:
```
With a comma, it looks like the rest of the sentence must be executed *in addition* to what precedes.
As I understand, the rest of the sentence is rather explaining *how* to do what precedes. Hence the proposal to use a colon instead.
A colon makes it more explicit than a comma, which may induce that the rest of the sentence